docs/examples/InputTypes.js

```diff
@@ -16,7 +16,9 @@ const inputTypeInstance = (
     </Input>
     <Input type='textarea' label='Text Area' placeholder='textarea' />
     <Input type='static' value='Static Text' />
-    <Input type='submit' value='Submit button' />
+    <ButtonInput value='Button Input' />
+    <ButtonInput type='reset' value='Reset Button' />
+    <ButtonInput type='submit' value='Submit Button' />
```

Comment:
I completely forgot about the children/value prop discussion we had last week about addressing the value. I'll need a bit.